app/controllers/introduce_yourself_controller.rb

```diff
@@ -17,6 +17,10 @@ def update
 
   private
 
+  def ensure_application_present
+    _application_not_created = true
+  end
```

Comment:
Instead of just placing `true` here I put the assignment to a descriptive variable instead. COULD delegate to another method but felt this was more direct and to the point. Up for discussion :)